src/Tokenizer.js

```diff
@@ -25,7 +25,7 @@ export function node (value, root, type, props, children, length) {
  * @param {string} type
  */
 export function copy (value, root, type) {
-	return node(value, root.root, type, root.props, root.children, 0)
+	return node(value, root.root, type, root.props, null, root.children, 0)
```

Comment:
Any reason we aren't carrying the root.parent reference over?

Reply:
I was thinking about 3 possibilities here:
1. carrying over root.parent (actually my initial choice)
2. null
3. pointing to the copied node - thought about it briefly, seemed cute at first but hard to tell apart the copied node from the original so I've discarded this quickly

In the end, I've chosen null - to keep `.parent` only for "real" nodes. Kinda similarly to how they always get `.length === 0` (regardless of the original value). I definitely see the value of carrying `.parent` over, so I'm totally OK with changing this.